docs/nidmm/functions.rst

```diff
@@ -1036,7 +1036,7 @@ nidmm.Session methods
 
 
 
-.. function:: get_dev_temp(options='')
+.. function:: get_dev_temp(options='""')
```

Comment:
Do these extra quotes cause any trouble?

Reply:
No, they are required since we use this string exactly. Other wise it would end up being option_string= instead of option_string=""